lib/action/sfAction.class.php

```diff
@@ -290,7 +290,7 @@ public function getPartial($templateName, $vars = null)
     {
         $this->getContext()->getConfiguration()->loadHelpers('Partial');
 
-        $vars = null !== $vars ? $vars : $this->varHolder->getAll();
+        $vars ??= $this->varHolder->getAll();
```

Comment:
this is not supported by php v7.4 IIRC

Reply:
https://wiki.php.net/rfc/null_coalesce_equal_operator